README.md

```diff
@@ -130,19 +126,15 @@ _ = manager.rx.responseString(.get, stringURL)
 _ = manager.rx.request(.get, stringURL)
     .validate(statusCode: 200 ..< 300)
     .validate(contentType: ["text/json"])
-    .flatMap {
-        $0.rx.string()
-    }
+    .string()
     .observeOn(MainScheduler.instance)
     .subscribe { print($0) }
 
 // URLHTTPResponse + Validation + URLHTTPResponse + String
 _ = manager.rx.request(.get, stringURL)
     .validate(statusCode: 200 ..< 300)
     .validate(contentType: ["text/json"])
-    .flatMap {
-        $0.rx.responseString()
-    }
+    .responseString()
```

Comment:
I'm confused - what's the difference between "string" and "responseString" ? 
Seems to be an Alamofire thing, but would love your explanation 😛

Reply:
Also, if you're validating a text/json header, maybe you should use responseJson here - would also be good to showcase we support that :)

Reply:
`responseString` = give me the http response (headers, status, etc) + the data as string
`string` = give me just the data as a string

Reply:
Cool, but as I asked - if you're validating the fact it has a JSON header, wouldn't it be more proper to use responseJson here?